vendor/k8s.io/klog/v2/go.mod

```diff
@@ -2,4 +2,7 @@ module k8s.io/klog/v2
 
 go 1.13
 
-require github.com/go-logr/logr v1.2.0
+require (
+	github.com/go-logr/logr v1.2.0
+	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
```

Comment:
Commented at https://github.com/kubernetes/klog/pull/293/files#r830100167, but this is introducing a circular dependency (utils -> klog -> utils). I don't think that's something we should do or pull into k8s.io/kubernetes

Reply:
Does it perhaps help that utils is only used by a test in klog?

Reply:
not really... circular dependencies easily lead to gridlock. if it's only used in a test, I'd push pretty hard to eliminate that dep (copy/paste the interface, etc)

Reply:
-> https://github.com/kubernetes/klog/pull/310